README.md

```diff
@@ -1 +1,9 @@
 # tarantool-qa
+
+Repository for QA activities and issues of the [tarantool/tarantool](https://github.com/tarantool/tarantool) project.
+
+Main target of this repository to detach internal QA activities from [Tarantool issues](https://github.com/tarantool/tarantool/issues) interesting to Tarantool users.
+This repository uses for issues on:
+ * Tests fixes for issues in tests.
+ * Tests issues reproducers.
+ * Setup test infrastructure, like [Github Actions](https://github.com/tarantool/tarantool/actions).
```

Comment:
I would adjust the wording a bit:

The repository is used for tasks of the following kinds:

* Testing problems (say, a badly written test). Logs, reproducers and so on are here.
* Testing infrastructure problems (say, network problem with some external resource).
* Testing enhancements and plans (for example, transition to [Github Actions](https://github.com/tarantool/tarantool/actions)).

Reply:
Right, I'll use your variant.

Reply:
Still 'This repository uses for issues on'. It looks grammatically incorrect: the repository *is used for*, not *uses for*.

Reply:
Corrected.